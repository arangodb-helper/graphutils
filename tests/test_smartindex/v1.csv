_key|Hello|smart
111|1|A
222|2|B
