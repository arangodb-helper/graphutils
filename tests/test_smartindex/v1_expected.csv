_key|Hello|smart
11:111|1|11
22:222|2|22
